### PR TITLE
Feature/yarn workspace

### PR DESCRIPTION
**Pull request by Pochodaydayup**
base #782

### CODE REVIEW

**Comment by yesmeck** [comment]
Site Preview: https://github.com/remaxjs/remax/pull/822

**Comment by yesmeck** [comment]
Site Preview: https://github.com/remaxjs/remax/pull/822

**Comment by yesmeck** [comment]
Site Preview: https://github.com/remaxjs/remax/pull/822

**Comment by yesmeck** [comment]
Site Preview: https://github.com/remaxjs/remax/pull/822

**Comment by yesmeck** [comment]
Site Preview: https://github.com/remaxjs/remax/pull/822

**Comment by yesmeck** [comment]
Site Preview: https://github.com/remaxjs/remax/pull/822

**Comment by yesmeck** [comment]
Site Preview: https://github.com/remaxjs/remax/pull/822

**Comment by yesmeck** [comment]
Site Preview: https://github.com/remaxjs/remax/pull/822

File: packages/remax-one/src/hostComponents/Input/props.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ interface CommonProps {
   disabled?: boolean;
   /** 最大输入长度，设置为 -1 的时候不限制最大长度 */
   maxLength?: number;
+  maxlength?: number;
```
Comment (yesmeck) [inline packages/remax-one/src/hostComponents/Input/props.ts:25]: 这个是冲突解决错了？

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+registry=https://registry.yarnpkg.com
```
Comment (yesmeck) [inline .npmrc:1]: 这个是必须的吗？

Comment (Pochodaydayup) [reply .npmrc:1]: 是的 还是加个吧 防止其他开发者贡献的时候吧 yarn.lock 的 registry 改了

File: packages/remax-one/src/hostComponents/Input/index.ts [hunk 3]
```diff
@@ -75,7 +75,7 @@ export default class Input extends React.Component<InputProps, InputState> {
     }
 
     if (process.env.REMAX_PLATFORM === 'wechat' || process.env.REMAX_PLATFORM === 'toutiao') {
-      inputProps.maxLength = inputProps.maxLength ?? 140;
```
Comment (Darmody) [inline packages/remax-one/src/hostComponents/Input/index.ts:78]: 这个不应该改掉

**Comment by yesmeck** [comment]
https://github.com/remaxjs/remax/blob/master/.github/workflows/test.yml#L23

这里要改成 `yarn install --no-lockfile`

File: packages/remax-one/src/hostComponents/Input/index.ts (hunk 3 above)
Comment (Pochodaydayup) [reply packages/remax-one/src/hostComponents/Input/index.ts:78]: 为啥 微信和头条上属性都是  `maxlength`

Comment (Darmody) [reply packages/remax-one/src/hostComponents/Input/index.ts:78]: 在 remax 里都是 `maxLength` ，然后会通过 属性别名对应到 `maxlength`

**Comment by Darmody** [comment]
LGTM

**Pull request merged**